galley/pkg/config/analysis/analyzers/analyzers_test.go

```diff
@@ -67,6 +67,27 @@ var testGrid = []testCase{
 			{msg.MisplacedAnnotation, "Deployment fortio-deploy"},
 		},
 	},
+	// {
```

Comment:
Accidentally left commented out?

Reply:
Whoops, good find. I accidentally commented this out as I was having problems building and running tests in istio 1.4 locally on my mac.

I may still need to fix some unit tests as the `ctx.ForEach(metadata.IstioAuthenticationV1Alpha1Policies, func(r *resource.Entry) bool {` statement was not actually looping (no auth policies were found).